### PR TITLE
Navbar & fetch profile from backend

### PR DESCRIPTION
**Pull request by anhfactor**
- Adjust navbar
- Fetch profile from backend api

https://user-images.githubusercontent.com/13186215/221623743-64efb6bd-2160-4b85-9fa9-30690145ae5e.mp4

### CODE REVIEW

**Comment by anhfactor** [comment]
- modify _app.js
- add _document.js
- add Loader & NotLoggedIn component

File: libs/api/src/common/config.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+{
+    CHAIN_NAME: "Optimism Goerli",
+    CHAIN_ID: "0x1a4",
+    NETWORK: "opt-goerli",
+    RPC_URL: "https://goerli.optimism.io",
+    L1_BLOCK_EXPLORER_URL: "https://goerli.etherscan.io",
+    L2_BLOCK_EXPLORER_URL: "https://goerli-optimism.etherscan.io",
+    LOGO_BADGE: "Goerli",
+    BACKEND_URL: "https://hd-backend-api.herokuapp.com",
+    BACKEND_TOKEN: "cc919b6a43b0921fc30fd3a6dbfdd9cdd636ffa9",
```
Comment (DistractionBoy) [inline libs/api/src/common/config.ts]: What are your plans for later? We don't wanna deploy with the token in the public repo, I assume

File: libs/ui/src/molecules/Navbar/Navbar.tsx [hunk 2]
```diff
@@ -73,11 +75,13 @@ const DarkNavbar = () => {
...
-                      <Image
-                        className="h-6 md:h-8 w-auto"
-                        src={logo}
-                        alt="Your Company"
-                      />
+                      <a href="/">
+                        <Image
+                          className="h-6 md:h-8 w-auto"
+                          src={logo}
+                          alt="Your Company"
```
Comment (DistractionBoy) [inline libs/ui/src/molecules/Navbar/Navbar.tsx]: Can you change this to "Hyperdrive"?

File: libs/ui/src/molecules/Navbar/Navbar.tsx [hunk 3]
```diff
@@ -145,4 +141,4 @@ const DarkNavbar = () => {
   );
 };
 
-export default DarkNavbar;
+export default Navbar;
```
Comment (DistractionBoy) [inline libs/ui/src/molecules/Navbar/Navbar.tsx:143]: 👍

File: libs/ui/src/molecules/NotLoggedIn/NotLoggedIn.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  getDefaultWallets,
+  RainbowKitProvider,
+  darkTheme
+} from '@rainbow-me/rainbowkit';
+import { configureChains, createClient, WagmiConfig } from 'wagmi';
+import { optimism, optimismGoerli } from 'wagmi/chains';
+import { alchemyProvider } from 'wagmi/providers/alchemy'
+import { publicProvider } from 'wagmi/providers/public';
+
+const alchemyId = process.env.NEXT_PUBLIC_ALCHEMY_ID || ''
```
Comment (DistractionBoy) [inline libs/ui/src/molecules/NotLoggedIn/NotLoggedIn.tsx]: Could you share this env var with the team in discord?

**Review by DistractionBoy — APPROVED** [review]
Great work!

File: libs/api/src/common/config.ts (hunk 1 above)
Comment (anhfactor) [reply libs/api/src/common/config.ts]: > What are your plans for later? We don't wanna deploy with the token in the public repo, I assume

let me change this, also create new token for test environment

**Pull request merged**